client.js

```
@@ -0,0 +1,202 @@
+            args = args.join(' ')
+
+            //get the open rcon to this server
+            let rcon = local_rcons[data.sever]
+
+            //wait 30 sec then /start the game with the arguments
+            setTimeout(async function() {
+                await rcon.send("/start " + args)
+            }, 30000)
+
+            //set the id to the data id
+            id = data.id
+
+            //break so default is not ran
+            break;
+        case 'lobby_set':
+            //lobby is the ip:port of the lobby server 
+            let lobby = data.data
+
+            //loop over all the local server and set the lobby
+            for (let name in servers.local_servers) {
+
+                //Get the open rcon connections
+                const rcon = local_rcons[name]
+
+                //Set the lobby
+                await rcon.send(`/interface global.servers = {lobby = '${lobby}'}`)
+                console.log(`${lobby} is the lobby.`)
+            }
+
+            //break so default is not ran
+            break;
+        default:
+            //If type is connected just print Connected to server
+            if (data.type === "connected") { console.log("Connected to server."); return }
+
+            //Else print that the type makes no sence
+            console.log("Unkown type " + data.type)
+            break
+    }
+}
+
+
+async function server_setup() {
+    //object for storing the games
+    var object_for_lua = {}
+
+    //check all local server
+    for (let variable in servers["local_servers"]) {
+
+        //Get the object from the locals servers
+        const object = servers["local_servers"][variable]
+
+        //Get the open rcon connection
+        const rcon = local_rcons[variable]
+
+        //Get the lobby key
+        const is_lobby = object.is_lobby
+
+        //Check if someting has goan wrong
+        if (is_lobby == true) { console.error('client cant have the lobby server'); return; }
```

This should throw an error.